lib/src/cli/arguments_builder/arguments_builder.dart

```diff
@@ -0,0 +1,59 @@
+import 'dart:io';
+
+import 'package:args/args.dart';
+
+import '../../analyzers/lint_analyzer/metrics/metrics_factory.dart';
+import '../../analyzers/lint_analyzer/metrics/models/metric_value_level.dart';
+import 'arguments_parser.dart';
+import 'arguments_validation.dart';
+import 'exceptions/arguments_validation_exceptions.dart';
+import 'models/flag_names.dart';
+import 'models/parsed_arguments.dart';
+
+// ignore: avoid_classes_with_only_static_members
```

Comment:
@incendial what's about rewrite this class without static methods?

Reply:
But why?